cypress/cypress/integration/lessons.spec.ts

```diff
@@ -97,11 +97,11 @@ describe("lessons screen", () => {
     cy.visit("/lessons");
     cy.get("#column-header");
     cy.get("#column-header #name").contains("Lesson");
-    cy.get("#column-header #launch").contains("Launch");
-    cy.get("#column-header #grade").contains("Grade");
+    // cy.get("#column-header #launch-button").contains("Launch");
```

Comment:
if we're getting rid of these, just delete...double check we actually don't need this coverage (e.g. there's a test for visibility of this button elsewhere?)

Reply:
Fixed